cmd/feed-ingress/main.go

```diff
@@ -41,7 +47,13 @@ func main() {
 	c := make(chan os.Signal, 1)
 	signal.Notify(c, os.Interrupt)
 
-	lb := ingress.NewLB()
+	lb := ingress.NewLB("/usr/sbin/nginx", nginxConfDir, nginxPort, &ingress.DefaultSignaller{})
```

Comment:
Maybe we should call it `NewNginxLB`. My idea is if we support other load balancers (like brocade), then it will just be a new constructor.